docs/content/how-to/openstack/az.md

```diff
@@ -0,0 +1,57 @@
+# Availability Zones
+
+This example will create a Nodepool that will spawn 5 machines in a given Nova availability zone.
+This can help to spread the workloads across multiple failure domains.
```

Comment:
> Availability Zones should not be assumed to map to fault domains and provide no intrinsic HA benefit by themselves.

From https://docs.openstack.org/nova/latest/admin/availability-zones.html

Not sure how you want to reword this.